.github/workflows/python-build.yml

```diff
@@ -0,0 +1,57 @@
+name: python-build
+
+env:
+  PROJECT_DIR: python
+
+on:
+  push:
+    paths:
+      - 'python/**'
+      - '.github/workflows/python-build.yml'
+  pull_request:
+    paths:
+      - 'python/**'
+      - '.github/workflows/python-build.yml'
+
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: ./${{ env.PROJECT_DIR }}
+
+    runs-on: ubuntu-22.04
+
+    env:
+      DB_USER: root
+      DB_OLD_PASSWORD: root
+      DB_PASSWORD: mysql
+      
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - name: Start MYSQL and import DB
```

Comment:
consider using https://github.com/marketplace/actions/actions-setup-mysql

Reply:
This part is just copied from the typescript version. Should probably update all the versions together?

Reply:
I would leave that for a separate PR

Reply:
I've found a cleaner way to do that:

https://github.com/nitsanavni/Refactoring-Kata-Lift-Pass-Pricing/pull/1/files#diff-88dc73ea041a254f6d251b3eb8872104b6926163e8a022e18a9b7d6939a99e73R23